src/Mainstore/Store.js

```diff
@@ -16,6 +17,7 @@ const reducers = combineReducers({
   signUpUser: signUpReducer,
   allArticleReducer,
   searchedArticles: SearchArticleReducer,
+  article: ViewArticlereducer,
```

Comment:
@jennizalwango I suggest you rename `article` to what relates to the reducer to avoid confusing it. Something like `viewAnArtricle`.

Reply:
okay noted let me work on it.

Reply:
kindly note l have worked on this